client/app/stories/stories.scss

```diff
@@ -0,0 +1,5 @@
+body {
+  background: none;
+  padding: 20px;
+  height: auto;
```

Comment:
Shouldn't the values for background and height be those values by default?

Reply:
Storybook has its own CSS styles, so we have to override them to achieve things like configurable background color. I think padding is just there to give space so that our components don't collide with the edge of the browser.